README.md

```diff
@@ -30,6 +30,7 @@ of the branches in the list.
 | `-m` `--modified`       | Show last modified branches |
 | `-i` `--no-interactive` | Don't use interactive mode. Interactive by default. |
 | `-c` `--count <number>` | Show number of branches. By default it shows nine.|
+| `--version`             | Show version number and quit |
```

Comment:
I think it's a matter of taste, for example `git` doesn't have a `-v` option, only a `--version` option.

If you think it's important I can add it.